spec/js-api/options.d.ts

```diff
@@ -60,8 +60,16 @@ export interface Options<sync extends 'sync' | 'async'> {
    *
    * [<ident-token>]: https://drafts.csswg.org/css-syntax-3/#ident-token-diagram
    *
-   * The compiler must throw an error if the `CustomFunction` does not return a
-   * `Value`.
+   * If the `CustomFunction` returns an invalid value, or a value that
+   * transitively contains an invalid value, the compiler must treat that as the
+   * Sass function throwing an error. The following values are considered
+   * invalid:
+   * 
```

Comment:
Not sure what this is complaining about, but maybe using dashes instead of asterisks for the bullet points below this list will fix it?